corosync_metrics.go

```diff
@@ -48,41 +119,42 @@ func parseQuoromStatus(quoromStatus []byte) (map[string]int, string, error) {
 	// check the case there is an sbd_config but the SBD_DEVICE is not set
 
 	if quorateWordPresent == "" {
-		return nil, "", fmt.Errorf("the quorum status output is not in parsable format as expected")
+		return nil, quorate, fmt.Errorf("the quorum status output is not in parsable format as expected")
 	}
 
 	quorateRaw := wordOnly.FindString(strings.SplitAfterN(quoromRaw, "Quorate:", 2)[1])
-	quorate := strings.ToLower(quorateRaw)
+	quorateString := strings.ToLower(quorateRaw)
+
+	if quorateString == "yes" {
+		quorate = 1
+	}
+
 	expVotes, _ := strconv.Atoi(numberOnly.FindString(strings.SplitAfterN(quoromRaw, "Expected votes:", 2)[1]))
 	highVotes, _ := strconv.Atoi(numberOnly.FindString(strings.SplitAfterN(quoromRaw, "Highest expected:", 2)[1]))
 	totalVotes, _ := strconv.Atoi(numberOnly.FindString(strings.SplitAfterN(quoromRaw, "Total votes:", 2)[1]))
 	quorum, _ := strconv.Atoi(numberOnly.FindString(strings.SplitAfterN(quoromRaw, "Quorum:", 2)[1]))
 
-	voteQuorumInfo := map[string]int{
-		"expectedVotes":   expVotes,
-		"highestExpected": highVotes,
-		"totalVotes":      totalVotes,
-		"quorum":          quorum,
+	quorumVotes = map[string]int{
+		"expected_votes":   expVotes,
+		"highest_expected": highVotes,
```

Comment:
I'm not sure if its golang convention to name keys of maps `foo_bar`.Afaik golang use `fooBar`

Reply:
is cosmetical in both direction but I would rather follow golang  convention of namings

Reply:
These keys maps 1:1 to the metric labels, so these are snake_cased to follow the Prometheus convention.
Having said that, I couldn't find any convention about map keys in the [Go style guide](https://github.com/golang/go/wiki/CodeReviewComments).

Reply:
yep let's ignore it for moment I'm ok